packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/CameraPropertiesImplTest.java

```diff
@@ -50,7 +50,7 @@ public void ctor_shouldReturnValidInstance() throws CameraAccessException {
   @SuppressWarnings("unchecked")
   public void getControlAutoExposureAvailableTargetFpsRangesTest() {
     Range<Integer> mockRange = mock(Range.class);
-    Range<Integer>[] mockRanges = new Range[] {mockRange};
+    Range<Integer>[] mockRanges = (Range<Integer>[]) new Range<?>[] {mockRange};
```

Comment:
Why is this a wildcard rather than `Integer`?

Reply:
Apparently the compiler doesn't like it. Added a comment to clarify.